test.py

```diff
@@ -0,0 +1,3 @@
+from sympy import *
+x = Symbol('x')
+print(gcd(x**2+1,x+I))
```

Comment:
I don't think that this file should be here

Reply:
Fixed